src/com/xilinx/rapidwright/rwroute/RWRoute.java

```diff
@@ -658,6 +658,9 @@ protected NetWrapper createNetWrapperAndConnections(Net net) {
                 }
                 if (sourceINTRnode == null && sourceINTNode != null) {
                     sourceINTRnode = getOrCreateRouteNode(sourceINTNode, RouteNodeType.PINFEED_O);
+                    // Always preserve the projected-to-INT source node, since it could
```

Comment:
```suggestion
                    // Where only a single primary source exists, always preserve 
                    // its projected-to-INT source node, since it could
```